hazelcast/src/main/java/com/hazelcast/jet/config/JetConfig.java

```diff
@@ -71,19 +281,29 @@ public JetConfig setResourceUploadEnabled(boolean resourceUploadEnabled) {
 
     /**
      * Returns the Jet instance config.
+     *
+     * @deprecated The fields from {@link InstanceConfig} class were
+     * moved to {@link JetConfig} class. Get the fields directly from
+     * {@link JetConfig}.
      */
     @Nonnull
+    @Deprecated
```

Comment:
There are still couple of usages of this method where we can stop using it.

Reply:
all the usages are explicit (parsing xml/yaml and tests)